Cython/Utility/Profile.c

```diff
@@ -193,6 +193,17 @@
   static PyCodeObject *__Pyx_createFrameCodeObject(const char *funcname, const char *srcfile, int firstlineno); /*proto*/
   static int __Pyx_TraceSetupAndCall(PyCodeObject** code, PyFrameObject** frame, PyThreadState* tstate, const char *funcname, const char *srcfile, int firstlineno); /*proto*/
 
+#if !CYTHON_USE_MODULE_STATE
+static PyObject *__pyx_dummy_tracer = NULL;
```

Comment:
```suggestion
static PyObject *$dummy_tracer = NULL;
```

The "subsitute: naming" option (which I believe is above) lets you get the name out of the `Naming` module rather than writing it explicitly in the utility code. 

There's a few more places in this PR with `__pyx_dummy_trace` which could be replaced with `$dummy_tracer`

Reply:
Will change it. Is there any documentation about the utility code transformations?

Reply:
> Is there any documentation about the utility code transformations?

Not really - there's a very small amount in the docstring in https://github.com/cython/cython/blob/fa8db66368a9527c8df441a0bbfee714eb795930/Cython/Compiler/Code.py#L221 but probably not enough to really be useful. The only other thing that I can think of (probably not relevant to this PR) is that `PYIDENT("some string")` and `PYUNICODE("some string")` turn the string into a module-level constant.

(My previous comment should have been `$dummy_tracer_cname` but it looks like you figured that out)